department-of-veterans-affairs/va.gov-cms
# VACMS-5372 add favicon

## Description

Relates to #5372 / ~#2746~. Also this related PR has been floating around since May: #5373 ... I created this new branch to avoid any potential merge conflicts with config / add the icon to the claro theme, but I could do this work in that PR too if that's more appropriate. I figured it could be safely closed instead :shrug:

- Adds favicon
- Removes environment_indicator addition to favicon
- Adds a screenshot for vagovclaro (seen on the appearance page)

## Testing done
Verified favicon showed as expected logged in and logged out with both vagovadmin and vagovclaro set as the default & admin themes.

## Screenshots
<img width="234" alt="Screen Shot 2021-12-21 at 2 52 15 PM" src="https://user-images.githubusercontent.com/11279744/147008292-7e31a971-b79a-4c88-a8e8-ee6619403e08.png">

<img width="1086" alt="Screen Shot 2021-12-21 at 11 35 19 AM" src="https://user-images.githubusercontent.com/11279744/147008263-10a13c14-2555-47c9-bc48-8c68e6827a29.png">


## QA steps
open PR environment, see that favicon is present
log in and change theme to vagovclaro, see that favicon is present for both themes

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

Personally I don't think an announcement is needed for a favicon update... just a nice quality of life update they may notice on their own?
